logshipper/tests/creation.tftest.hcl

```
@@ -0,0 +1,46 @@
+provider "cloudfoundry" {
+  api_url = "https://api.fr.cloud.gov"
+  # cf_user and cf_password are passed in via CF_USER and CF_PASSWORD env vars
+}
```

[suggestion] we can use cf-cli auth in tests too if we remove all config set here.

```suggestion
provider "cloudfoundry" {}
```

---

Fixed